quickstart.md

```diff
@@ -513,6 +513,7 @@ pod_ip=$(kubectl get pod -o wide | grep encrypted-image-tests | awk '{print $6;}
 Get the CoCo sample encrypted container image SSH access key from docker image label and save it to a file:
 
 ```
+sudo docker pull quay.io/kata-containers/encrypted-image-tests:encrypted
```

Comment:
While we are here, seems like we could (should?) be doing `docker` without `sudo`?